dash-pipeline/tests/libsai/vnet_out/vnet_out.cpp

```diff
@@ -48,7 +60,7 @@ int main(int argc, char **argv)
     QUERY_STATUS_CHECK(status, SAI_API_DASH_ENI);
 
     sai_direction_lookup_entry_t dle = {};
-    dle.switch_id = switch_id;
+    dle.switch_id = 0; // TODO switch_id, it fails when switch id is non zero
```

Comment:
@marian-pritsak @chrispsommers, in this PR i added check, that switch must be created first before creating any other object, but somehow this fails if switch_id is non zero, mutate table entry fails, this comes from deep bmv2/p4 internals, not sure why this happens

Reply:
@kcudnik It'd be helpful to include a console dump of bmv2 model for this failure.

Reply:
hmm, locally it passes, but here it failed: https://github.com/sonic-net/DASH/actions/runs/6174596066/job/16759546592?pr=438#step:13:25

Reply:
testing again

Reply:
false alarm